internal/criapi/criapi_test.go

```diff
@@ -0,0 +1,43 @@
+//+build k8s
```

Comment:
Needs copyright header?

Reply:
Fixed in next push. Note- compiler complains if the //+build k8s line and following blank line are after the Copyright comment with "./criapi_test.go:12:1: +build comment must appear before package clause and be followed by a blank line".